src/Tutor.Web/Controllers/Plugin/PluginController.cs

```diff
@@ -16,12 +18,14 @@ public class PluginController : ControllerBase
         private readonly ILearnerRepository _learnerRepository;
         private readonly IMapper _mapper;
         private readonly ISubmissionService _submissionService;
+        private readonly IAssessmentEventHelpService _interactionService;
 
-        public PluginController(ILearnerRepository learnerRepository, ISubmissionService submissionService, IMapper mapper)
+        public PluginController(ILearnerRepository learnerRepository, IMapper mapper, ISubmissionService submissionService, IAssessmentEventHelpService interactionService)
         {
             _learnerRepository = learnerRepository;
-            _submissionService = submissionService;
             _mapper = mapper;
+            _submissionService = submissionService;
+            _interactionService = interactionService;
```

Comment:
interactionService is a way too generic name.

Reply:
Fixed, I accidentally forgot to change that.